partner_communication_switzerland/views/recurring_contracts_view.xml

```diff
@@ -9,4 +9,62 @@
             </field>
         </field>
     </record>
-</odoo>
\ No newline at end of file
+    <record id="invoice_line_inherit_muskathlon" model="ir.ui.view">
```

Comment:
All those muskathlon records have been moved to the module `muskathlon`. You can remove them